cmake/define-ia2-wrapper.cmake

```diff
@@ -166,6 +166,7 @@ function(define_ia2_wrapper)
           -D_GNU_SOURCE
           -isystem ${C_SYSTEM_INCLUDE}
           -isystem ${C_SYSTEM_INCLUDE_FIXED}
+          -I ${REWRITTEN_HEADER_DIR}
```

Comment:
We can't do this because libavutil includes `time.h` which must not shadow the system header of the same name.

Reply:
So this should be the root build dir?